File: net/API/Models/SignalEvent.cs
namespace API.Models
{
    public class SignalEvent
    {
        // TODO follow naming conventions
        public string id_sample { get; set; } // item identifier (76rtw)
        public string num_id { get; set; } // item serial number (ffg#er111)
        public string id_location { get; set; } // location from (3211.2334), can be a name
        public string id_signal_par { get; set; } // sensor generating signal (0xcv11cs)
        public string id_detected { get; set; } // status data (None), - functional, (Nan), - failed
        public string id_class_det { get; set; } // failure type (req11)
    }
}

File: java/api/src/main/java/sinologic16/api/model/SignalEvent.java
package sinologic16.api.model;

import lombok.Getter;
import lombok.Setter;


@Getter
@Setter
public class SignalEvent {
    public String id_sample; // item identifier (76rtw)
    public String num_id; // item serial number (ffg#er111)
    public String id_location; // location from (3211.2334), can be a name
    public String id_signal_par; // sensor generating signal (0xcv11cs)
    public String id_detected; // status data (None), - functional, (Nan), - failed
    public String id_class_det; // failure type (req11)
}